packages/react-bindings/test/styles/convertCssTimeToNumber-test.ts

```diff
@@ -0,0 +1,25 @@
+import { unstable_calculateAnimationTimeout as calculateAnimationTimeout } from '@fluentui/react-bindings'
+
+describe('convertCssTimeToNumber', () => {
+  test('converts ms values to number', () => {
+    expect(calculateAnimationTimeout('200ms', '300ms')).toEqual(500)
+  })
+
+  test('converts s values to number', () => {
+    expect(calculateAnimationTimeout('20s', '2s')).toEqual(22000)
+  })
+
+  test('converts other valid css values to 0', () => {
+    expect(calculateAnimationTimeout('initial', '200ms')).toEqual(0)
+    expect(calculateAnimationTimeout('inherit', '400ms')).toEqual(0)
```

Comment:
Is this valid? Should it be `400ms`?

Reply:
If there is `initial` or `inherit`, we should return 0, as we cannot calculate it, and the user should provide the timeout value, or we will remove/add the element immediately.